platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -128,6 +128,14 @@ If only warnings happen (and no `-Werror` is set), this report will not be visib
 
 This feature also works with the [`--continue`](userguide/command_line_interface.html#sec:continue_build_on_failure) flag, and the bottom report will contain a per-task report of all the compilation failures.
 
+#### HTML problems report
+
+All problems reported via the [Problems API](userguide/implementing_gradle_plugins_binary.html#reporting_problems), internally in Gradle or from a plugin, will be rendered into an HTML problems report
+by default.
```

Comment:
Is it possible to show an example console output that showcases the HTML report?